electron/main.js

```diff
@@ -169,13 +169,13 @@ function showMainWindow() {
     main.setBounds(init.restore('bounds', main.getBounds()));
   }
 
-  main.webContents.session.setCertificateVerifyProc((hostname = '', certificate = {}, cb) => {
+  main.webContents.session.setCertificateVerifyProc(({hostname = '', certificate = {}}, cb) => {
     const {data: cert = ''} = certificate;
```

Comment:
Now that `request` is an object, we can use 
```js
const { hostname = '', certificate: { data: cert = '' } = {}, error } = request;
```
so we don't destructure twice. I think it's more consistent.

And check for error maybe?